github/ci/prow/files/jobs/kubevirt/user-guide/user-guide-periodics.yaml

```diff
@@ -0,0 +1,23 @@
+periodics:
+  - interval: 24h
+    name: user-guide-periodic-build
+    decorate: true
+    extra_refs:
+      - org: kubevirt
+        repo: user-guide
+        base_ref: source
+        path_alias: user-guide
+    spec:
+      nodeSelector:
+        region: primary
+      containers:
+        - image: docker.io/library/python3
+          command: ["/bin/sh", "-c"]
+          args: ["echo &&
+                  mkdir -p /opt/buildhome/python3 &&
+                  source /opt/buildhome/python3/bin/activate &&
+                  pip3 install mkdocs mkdocs-awesome-pages-plugin &&
+                  echo '*** BEGIN mkdocs.yml ***' &&
+                  cat /opt/build/repo/mkdocs.yml &&
+                  echo '*** END mkdocs.yml ***' &&
+                  mkdocs build -f mkdocs.yml -d site"]
```

Comment:
IIUC the mkdocs documentation mkdocs is not directly supported as Jekyll is for GH Pages, you need to invoke the step `mkdocs gh-deploy` for this to happen. Or does this happen elsewhere?